sdk/src/AbstractMain.cpp

```diff
@@ -230,14 +232,17 @@ int AbstractMain::run(int in_argc, char** in_argv)
 
    if (options.getLogLevel() > LogLevel::INFO)
    {
-      addLogDestination(
+       
+        addLogDestination(
          std::unique_ptr<ILogDestination>(
             new FileLogDestination(
-               3,
+               "3",
```

Comment:
```suggestion
               "core::system::generateShortenedUuid()",
```

Giving the log destination a unique ID